src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -289,22 +288,27 @@ private StripeError parseStripeError(
   }
 
   private void handleError(StripeResponse response, ApiMode apiMode) throws StripeException {
-    JsonObject responseBody = ApiResource.GSON.fromJson(response.body(), JsonObject.class);
-
-    /*
-     OAuth errors are JSON objects where `error` is a string. In
-     contrast, in API errors, `error` is a hash with sub-keys. We use
-     this property to distinguish between OAuth and API errors.
-    */
-    if (responseBody.has("error") && responseBody.get("error").isJsonPrimitive()) {
-      JsonPrimitive error = responseBody.getAsJsonPrimitive("error");
-      if (error.isString()) {
-        handleOAuthError(response);
+    try {
+      /*
+      OAuth errors are JSON objects where `error` is a string. In
+      contrast, in API errors, `error` is a hash with sub-keys. We use
+      this property to distinguish between OAuth and API errors.
+
+      Try to read the response body to see if it must be
```

Comment:
Is this missing the second part of the sentence?

Reply:
My mistake, fixed